apps/catalog/catalog/src/app/marketplace/offer/shell/contract-list/contract-list.component.scss

```
@@ -3,7 +3,11 @@
   margin: 36px 48px;
 }
 
-bf-filter-table {
+bf-table {
   display: block;
   width: 100%;
 }
+
+h3 {
+  margin: 24px 0 16px 0;
+}
```

As much as possible avoid asymmetric margin/padding. If you cannot separate them for readability :
```suggestion
h3 {
  margin-top: 24px;
  margin-bottom: 16px;
}
```